frontend/src/app/features/details/summary/SummaryInfo.tsx

```diff
@@ -33,14 +50,57 @@ const SummaryInfo: React.FC<ISummaryInfo> = ({
       console.log('Approve Handler Not Provided');
     });
 
+  const CENTER_OF_BC: LatLngTuple = [53.7267, -127.6476];
+
+  const { data } = useMapSearch_FindSiteBySiteIdQuery({
+    variables: { siteId: siteData.id },
+  });
+
+  const selectedSiteData = data?.findSiteBySiteId.data;
+  const mapRef = useRef<Map>(null);
+  const map_fly_options = { animate: false, duration: 0.2 };
+
+  const MapWithMarker = ({ site }: { site: any }) => {
+    const map = useMap();
+
+    useEffect(() => {
+      if (!site.latdeg || !site.longdeg) return;
+      map.flyTo(
+        {
+          lat: site.latdeg,
+          lng: site.longdeg,
+        },
+        getZoom(map),
+        map_fly_options,
+      );
+    }, [site, map]);
+
+    return (
+      site && <SiteMarker position={{ lat: site.latdeg, lng: site.longdeg }} />
+    );
+  };
+
   return (
     <PanelWithUpDown
       label="Location Details"
       secondChild={
         <div className="row w-100">
           <div className="col-12 col-lg-6">
-            {/* <Map callback={() => {}} initLocation={location} readOnly={true} /> */}
+            <MapContainer
+              center={CENTER_OF_BC}
```

Comment:
How about we just do this?

```tsx
<MapContainer
    center={{ lat: siteData.latdeg, lng: siteData.longdeg }}
```

We're not doing any animations on load and already have the lat/deg data at this point.